.gitignore

```diff
@@ -0,0 +1,11 @@
+.DS_Store
+wp-cli.local.yml
```

Comment:
We can remove this now from all repo. We don't have wp-cli.local.yml. Maybe check if we can have ee-cli.local.yml.